rel/env.sh.eex

```diff
@@ -0,0 +1,7 @@
+# release always starts in a distributed mode
+# but to provide unified way of configuring Jellyfish
+# distribution both in the local and production environment
+# we introduce our own env vars and use them to
+# set RELASE_NODE and RELASE_COOKIE
```

Comment:
```suggestion
# We introduce our own env vars and use them to set
# RELEASE_NODE and RELEASE_COOKIE.
#
# This is to provide a unified way of configuring Jellyfish distribution
# in both development and production environments
```

Reply:
The info that release always starts in a distributed mode is crucial here